src/test/java/core/basesyntax/service/handlers/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.service.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Warehouse;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
```

Comment:
OperationStrategyImpl test should test work of methods of OperationStrategyImpl class(get(Transaction.Operation operation) method), and here you test work of handle method. So we should create separate test classes for every handler